cluster-autoscaler/dynamic/config.go

```diff
@@ -0,0 +1,86 @@
+package dynamic
```

Comment:
After reading the contents of that package it doesn't seem like its being used cluster autoscaler config anyways.. maybe it should but might be outside the scope of this PR. I'll let the maintainers decide :P